modules/ROOT/pages/until-successful-scope.adoc

```diff
@@ -7,7 +7,7 @@ endif::[]
 The Until Successful scope processes the components within it, in order, until they
 succeed or exhaust the maximum number of retries. Like all Core components
 other than Async, Until Successful runs synchronously. If a component
-within the scope fails to connect or produce a successful result,
+within the scope fails to connect or produce an unsuccessful result,
```

Comment:
The phrase should be read as "fails to connect or fails to produce a successful result".